concord-server/server/src/test/java/com/rabidgremlin/concord/functions/GetDeadLockedPhrasesFunctionTest.java

```diff
@@ -211,4 +211,25 @@ public void shouldIgnorePhrasesWhichHaveOnlyOneLabelVotedOn()
     assertThat(deadLockedPhrases.size(), is(0));
   }
 
+  @Test
+  public void shouldAccountForFilteredUsersVotes()
+  {
+    // Given
+    // 7 users but 8 votes (BULK_UPLOAD has voted)
+    final int userCount = 7;
+    final int consensusLevel = 3;
+    List<GroupedPhraseVoteWithMostRecentVoteTime> phraseVotes = new ArrayList<>();
+
+    phraseVotes.add(new GroupedPhraseVoteWithMostRecentVoteTime("BonJovi", "Woah, livin' on a prayer", 5, TIME_STAMP));
+    phraseVotes.add(new GroupedPhraseVoteWithMostRecentVoteTime("Beyonce", "Woah, livin' on a prayer", 2, TIME_STAMP));
+    phraseVotes.add(new GroupedPhraseVoteWithMostRecentVoteTime("Elvis", "Woah, livin' on a prayer", 1, TIME_STAMP));
```

Comment:
We should have a unit test that verifies that inactive users are not counted.

Reply:
can't test since I did the filter in the SQL.
would need to mock the database tables.